cmd/spire-server/cli/federation/refresh.go

```diff
@@ -44,21 +43,16 @@ func (c *refreshCommand) Run(ctx context.Context, env *common_cli.Env, serverCli
 		return errors.New("id is required")
 	}
 
-	id, err := idutil.NormalizeSpiffeID(c.id, idutil.AllowAnyTrustDomain())
-	if err != nil {
-		return err
-	}
-
 	trustDomainClient := serverClient.NewTrustDomainClient()
-	_, err = trustDomainClient.RefreshBundle(ctx, &trustdomain.RefreshBundleRequest{
-		TrustDomain: id,
+	_, err := trustDomainClient.RefreshBundle(ctx, &trustdomain.RefreshBundleRequest{
+		TrustDomain: c.id,
 	})
 	switch status.Code(err) {
 	case codes.OK:
 		env.Println("Bundle refreshed")
 		return nil
 	case codes.NotFound:
-		return fmt.Errorf("there is no federation relationship with trust domain %q", id)
+		return fmt.Errorf("there is no federation relationship with trust domain %q", c.id)
```

Comment:
That is correct. We don't want the CLI layer munging or otherwise interpreting the user input, instead deferring to the APIs to decide what is correct or not.